.github/workflows/build_image.yaml

```diff
@@ -6,13 +6,16 @@ on:
     - '**'
 
 env:
+  DOCKER_CACHE_FROM: docker.io/timescale/timescaledb-ha:pg14-builder
```

Comment:
sorry for the noise, I sorted the list to make for easier comparison in the future